### PR TITLE
add lend page

### PR DESCRIPTION
**Pull request by 3mp8r3**
- added lend page from zeplin

### CODE REVIEW

File: src/components/CustomSelect/index.tsx [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+    };
+  }, []);
+  useEffect(() => {
+    const list: any = [];
+    if (!Array.isArray(children)) {
+      setOptionList(list);
+      return;
+    }
+    children?.map((each, index) => {
+      if (each.props.customTagType !== '--411customisedOption--') {
```
Comment (FriskyHamTitz) [inline src/components/CustomSelect/index.tsx:52]: Leave a comment explaining --411

File: src/components/CustomSelect/index.tsx [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+        >
+          {before}
+          {currentOption?.text}
+          <Box ml={'auto'} display={'flex'} alignItems={'center'}>
+            <NarrowIcon dir={isOpened ? 'up' : 'down'} />
+          </Box>
+          {after}
+        </Box>
+      )}
+      <DropPannel
```
Comment (FriskyHamTitz) [inline src/components/CustomSelect/index.tsx:140]: DropPannel should be named DropPanel

File: src/components/SearchInput/SearchInput.tsx [hunk 3]
```diff
@@ -9,11 +9,11 @@ const useStyles = makeStyles(({ palette, breakpoints }) => ({
     height: 40,
     border: `1px solid ${palette.secondary.dark}`,
     borderRadius: 10,
-    minWidth: 300,
+    // minWidth: 300,
```
Comment (FriskyHamTitz) [inline src/components/SearchInput/SearchInput.tsx:12]: Do not leave commented out code in the source that's what source control is for

File: src/pages/LendPage/LendDetailPage.tsx [hunk 4]
```diff
@@ -0,0 +1,450 @@
...
+          sx={{ minWidth: { xs: '55%', sm: '35%', md: '20%' } }}
+          bgcolor={'#232734'}
+          p={'24px'}
+          borderRadius={'12px'}
+        >
+          <Box color={'#696c80'} fontSize={'14px'}>
+            Total Borrowed
+          </Box>
+          <Box fontSize={'24px'} color={'white'}>
+            $19.2M
```
Comment (FriskyHamTitz) [inline src/pages/LendPage/LendDetailPage.tsx:116]: Why is 19.2m hardcoded?

File: src/pages/LendPage/LendDetailPage.tsx [hunk 5]
```diff
@@ -0,0 +1,450 @@
...
+          sx={{ minWidth: { xs: '55%', sm: '35%', md: '20%' } }}
+          bgcolor={'#232734'}
+          p={'24px'}
+          borderRadius={'12px'}
+        >
+          <Box color={'#696c80'} fontSize={'14px'}>
+            Pull Utilization
+          </Box>
+          <Box fontSize={'24px'} color={'white'}>
+            $19.2M
```
Comment (FriskyHamTitz) [inline src/pages/LendPage/LendDetailPage.tsx:144]: Why is this hardcoded?

File: src/pages/LendPage/LendPage.tsx [hunk 6]
```diff
@@ -0,0 +1,435 @@
...
+              // isMobile ? 'Search' : 'Search name, symbol or paste address'
+              'Search'
+            }
+            value={searchInput}
+            setValue={setSearchInput}
+          />
+          <CustomSelect width={'calc(50% - 8px)'}>
+            <SmOption>SortBy: Rewards</SmOption>
+            <SmOption>12313123123</SmOption>
+            <SmOption>12313123123</SmOption>
```
Comment (FriskyHamTitz) [inline src/pages/LendPage/LendPage.tsx:131]: 123123123????

**Review by FriskyHamTitz — CHANGES_REQUESTED** [review]

File: src/components/SearchInput/SearchInput.tsx (hunk 3 above)
Comment (3mp8r3) [reply src/components/SearchInput/SearchInput.tsx]: removed commented codes

File: src/pages/LendPage/LendDetailPage.tsx (hunk 4 above)
Comment (3mp8r3) [reply src/pages/LendPage/LendDetailPage.tsx:116]: updated the value on top of zeplin mocup.

File: src/pages/LendPage/LendDetailPage.tsx (hunk 5 above)
Comment (3mp8r3) [reply src/pages/LendPage/LendDetailPage.tsx:144]: updated the value on top of zeplin mockup.

File: src/pages/LendPage/LendPage.tsx (hunk 6 above)
Comment (3mp8r3) [reply src/pages/LendPage/LendPage.tsx]: fixed the options based on pool names for now.

File: src/components/CustomSelect/index.tsx (hunk 2 above)
Comment (3mp8r3) [reply src/components/CustomSelect/index.tsx]: updated DropPanel names

File: src/components/CustomSelect/index.tsx (hunk 1 above)
Comment (3mp8r3) [reply src/components/CustomSelect/index.tsx:52]: add comment about customTagType

File: src/components/CustomSelect/index.tsx [hunk 7]
```diff
@@ -0,0 +1,251 @@
...
+                    if (currentOption.value === each.value) return;
+                    setCurrentOption(each);
+                    onChange && onChange(each.value);
+                  }}
+                >
+                  {each.text}
+                </SmOption>
+              );
+            })
+          : 'Here is not any options.'}
```
Comment (FriskyHamTitz) [inline src/components/CustomSelect/index.tsx:173]: "Here is not any options" should say "there are no options available"

File: src/pages/LendPage/LendDetailPage.tsx [hunk 8]
```diff
@@ -0,0 +1,1228 @@
...
+      >
+        <Box
+          onClick={() => {
+            history.push('../lend');
+          }}
+          display={'flex'}
+          alignItems={'center'}
+          style={{ cursor: 'pointer' }}
+        >
+          <svg
```
Comment (FriskyHamTitz) [inline src/pages/LendPage/LendDetailPage.tsx:49]: Why aren't these svgs in an svg file in assets

File: src/pages/LendPage/LendDetailPage.tsx [hunk 9]
```diff
@@ -0,0 +1,1228 @@
...
+          sx={{ minWidth: { xs: '55%', sm: '35%', md: '20%' } }}
+          bgcolor={'#232734'}
+          p={'24px'}
+          borderRadius={'12px'}
+        >
+          <Box color={'#696c80'} fontSize={'14px'}>
+            Total Supply
+          </Box>
+          <Box fontSize={'24px'} color={'white'}>
+            $19.2M
```
Comment (FriskyHamTitz) [inline src/pages/LendPage/LendDetailPage.tsx:116]: I thought we were going to obtain these values from somewhere?

**Review by FriskyHamTitz — CHANGES_REQUESTED** [review]
My issue with the PR is that we're using mock values, why aren't we obtaining them from the underlying contracts or an API?

File: src/components/Modals/index.tsx [hunk 10]
```diff
@@ -0,0 +1,606 @@
...
+interface IconProps {
+  size?: any;
+  color?: any;
+}
+const LogoIcon: React.FC<IconProps> = ({
+  size = '1em',
+  color = 'currentColor',
+}) => {
+  return (
+    <svg
```
Comment (FriskyHamTitz) [inline src/components/Modals/index.tsx:545]: Svgs should be stored in their own files.

**Pull request closed without merging**